data/json/items/armor/shields.json

```
@@ -161,5 +161,73 @@
     "bashing": 10,
     "encumbrance": 30,
     "techniques": [ "WBLOCK_3" ]
+  },
+  {
+    "id": "shield_welded",
+    "type": "ARMOR",
+    "name": { "str": "welded shield" },
+    "description": "A shield made out of welded sheet metal.  It's a lot heavier and less well-made compared to a proper medieval shield, but it certainly works.",
+    "weight": "8000 g",
+    "volume": "5 L",
+    "price_postapoc": "5 USD",
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel" ],
+    "looks_like": "shield_ballistic",
+    "symbol": "[",
+    "color": "blue",
+    "covers": [ "arm_either", "hand_either" ],
+    "coverage": 90,
+    "encumbrance": 40,
+    "material_thickness": 3,
+    "techniques": [ "WBLOCK_3" ],
+    "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "BLOCK_WHILE_WORN" ]
+  },
+  {
+    "id": "shield_buckler",
+    "copy-from": "shield_banded",
+    "looks_like": "shield_banded",
+    "type": "ARMOR",
+    "name": { "str": "buckler" },
+    "description": "A small metal shield historically used for dueling rather than protection from arrows.  It's small and light enough to not hinder the use of your hands.",
+    "volume": "1 L",
+    "weight": "1400 g",
+    "price": "120 USD",
+    "price_postapoc": "10 USD",
+    "material": [ "steel" ],
+    "to_hit": 0,
+    "covers": [ "arm_either" ],
+    "coverage": 50,
+    "encumbrance": 10,
+    "flags": [ "OVERSIZE", "BELTED", "ONLY_ONE", "BLOCK_WHILE_WORN", "STURDY" ]
+  },
+  {
+    "id": "shield_spiked",
+    "copy-from": "shield_banded",
+    "looks_like": "shield_banded",
+    "type": "ARMOR",
+    "name": { "str": "spiked shield" },
+    "description": "A metal reinforced shield with spikes jutting out of it.  Can be used as a melee weapon if need be.",
+    "weight": "3500 g",
+    "price": "100 USD",
+    "price_postapoc": "12 USD",
+    "to_hit": 0,
+    "bashing": 22,
+    "cutting": 22,
```

For now I'd probably lean towards just making this less hefty, and hopefully I or someone else can figure out how to add spiked effects at some point. Will need to be tied to the item actually being used to block with and/or triggering damage reduction as armor. Also it probably shouldn't gain massively elevated stats over its standard version, so probably okay to just keep the same bash value and to-hit, and add cutting damage that matches.
```suggestion
    "cutting": 8,
```